src/Tribe/Editor/Configuration.php

```diff
@@ -46,6 +46,13 @@ public function editor_config( $editor_config ) {
 			$this->localize()
 		);
 
+		/**
+		 * Filter the default buffer duration between ticket sale start time and end time.
+		 *
+		 * @param int $buffer Number in hours to be used.
+		 */
+		$editor_config['tickets']['end_sale_buffer_duration'] = apply_filters( 'tribe_ticket_end_sale_buffer_duration_hours', 2 );
+
```

Comment:
```suggestion

		/**
		 * Filter the default buffer years between ticket sale start time and end time.
		 *
		 * @since TBD
		 *
		 * @param int $buffer Number in hours to be used.
		 */
		$editor_config['tickets']['end_sale_buffer_years'] = apply_filters( 'tribe_ticket_editor_end_sale_buffer_years', 1 );

```